irods_specific_implementation/irods_rest_list_api_implementation.h

```diff
@@ -49,9 +49,9 @@ namespace irods::rest {
                 const intmax_t offset = std::stoi(_offset);
                 const intmax_t limit  = std::stoi(_limit);
 
-                const bool stat        = ("true" == _stat);
-                const bool permissions = ("true" == _permissions);
-                const bool metadata    = ("true" == _metadata);
+                const bool stat        = ("true" == _stat || "1" == _stat);
+                const bool permissions = ("true" == _permissions || "1" == _permissions);
+                const bool metadata    = ("true" == _metadata || "1" == _metadata);
```

Comment:
Let's move to using only 1's and 0's. Standardize on a single value rather than allowing multiple values.

Reply:
Couldn't removing "true" from these lines potentially break some already-existing code?

Reply:
Yes, that is correct - good thinking/defense.

However, since we're still at 0.9.1... I think we're considering this a documentation exercise as the interface stabilizes.

I agree - let's settle on just 0/1 for these flags.